tests/modules/transdecoder/longorf/test.yml

```diff
@@ -0,0 +1,22 @@
+- name: transdecoder longorf test_transdecoder_longorf
+  command: nextflow run tests/modules/transdecoder/longorf -entry test_transdecoder_longorf -c tests/config/nextflow.config
+  tags:
+    - transdecoder
+    - transdecoder/longorf
+  files:
+    - path: output/transdecoder/pipeliner.3429110.cmds
+      md5sum: abf07276142f8372228e799ae746ab06
+    - path: output/transdecoder/test.__checkpoints_longorfs/TD.longorfs.ok
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/transdecoder/test.__checkpoints_longorfs/base_freqs_file.ok
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
As I've commented for other files, this should probably not be output as a channel.